src/database/seeders/20191028063855-demo-vendor.js

```diff
@@ -1,113 +1,110 @@
 import uuidv4 from 'uuid/v4';
-import bcrypt from 'bcrypt';
+import bcrypt from 'bcryptjs';
```

Comment:
Why `bcryptjs` instead of `bcrypt` that we have before?

Reply:
Sandra, you probably have issues running bcrypt on your laptop, I did too. What you'll have to do is:
1. Install windows-build-tools and python2.7
2. Install Visual Studio with C++ worktools
That's how I got around the issue